status/keyactions.go

```diff
@@ -44,7 +44,7 @@ type CreateNewEncryptionSubkey struct {
 }
 
 func (a CreateNewEncryptionSubkey) Enact(key *pgpkey.PgpKey, now time.Time) error {
-	return key.CreateNewEncryptionSubkey(a.ValidUntil, now)
+	return key.CreateNewEncryptionSubkey(a.ValidUntil, now, nil)
```

Comment:
That's interesting. So passing nil to `packet.Config{Rand}` means it uses a cryptographically random number?

Reply:
I've updated the docs